test/controllers/graphql_controller_3_test.rb

```diff
@@ -188,7 +188,7 @@ def setup
     pm1b = create_project_media project: p1b, disable_es_callbacks: false ; sleep 1
     pm1b.disable_es_callbacks = false ; pm1b.updated_at = Time.now ; pm1b.save! ; sleep 1
     pm1a.disable_es_callbacks = false ; pm1a.updated_at = Time.now ; pm1a.save! ; sleep 1
-    pm1c = create_project_media project: p1a, disable_es_callbacks: false, archived: true ; sleep 1
+    pm1c = create_project_media project: p1a, disable_es_callbacks: false, archived: 1 ; sleep 1
```

Comment:
Why not use the constants in the test too?

Reply:
Fixed by using constants